rero_ils/modules/stats/serializers.py

```diff
@@ -75,6 +137,38 @@ def _format_csv(self, records):
                 value = StatCSVSerializer.sort_dict_by_key(value)[1]
                 writer.writerow(value)
                 yield line.read()
+        elif record['metadata'].get('type') == StatType.REPORT:
+            # statistics of type report
+            indicator = record['metadata']['config']['category']['indicator']
+            distributions = indicator.get('distributions')
+            results = True
+
+            # case report with no query results
+            if len(record['metadata']['values']) == 1\
+               and record['metadata']['values'][0][indicator['type']] == 0:
+                results = False
+
+            # case report with 2 distributions
+            if distributions and len(distributions) == 2 and results:
```

Comment:
... you could remove `if distributions` condition :: `if len(distributions) == 2 and results:`